### PR TITLE
Update library spec to remove classes

### LINKED ISSUE #581: Update first solo project 2
# Project to be updated: [JS Fun at the library](https://frontend.turing.edu/projects/module-1/library.html)

## Summary of changes needed: 
- This project currently asks students to implement a Class (Librarian) based on a test suite
- The project needs to be updated so the the tests are not expecting a Class to be created
- Librarian is an extension for the project, so a different extension should replace it

### PR DESCRIPTION
**Pull request by kaylagordon**
I mirrored @Kalikoze's work on JS Fun at the Restaurant as closely as possible. 
- The class extension is gone (`librarian`)
- A refactoring extension is added
- One additional function is added to `library`

PR for the repo: https://github.com/turingschool-examples/js-fun-at-the-library/pull/33

Note that I'm merging directly into `gh-pages` due to the time constraints on this particular update. This needs to be merged by 3/21.

closes #581

### CODE REVIEW

File: projects/module-1/library.md [hunk 1]
```diff
@@ -82,11 +82,11 @@ problem.
     different way you could make the tests pass?
 
 
-### Extension (not part of the project, just stuff to do if you get to it): Complete the Librarian tests
-  - Unskip the first test in `test/librarian-test.js`, and get to work passing the tests
-  - Run `npm test test/librarian-test.js`  
-  - This test involves using a class. It will require self-teaching in order to complete.
-  - Ensure that all of the skips are removed from the test file when you push up to GitHub.
+### Extension: Refactoring
```
Comment (letakeane) [inline projects/module-1/library.md:85]: Love the focus on refactoring!

File: projects/module-1/library.md [hunk 2]
```diff
@@ -82,11 +82,11 @@ problem.
...
+  - Preteach yourself the [Dot & Bracket Notation](https://frontend.turing.edu/lessons/module-1/js-dot-bracket-notation.html){:target="_blank"} lesson.
```
Comment (letakeane) [inline projects/module-1/library.md:86]: Such a meaningful add :)

**Review by letakeane — APPROVED** [review]
Spec updates look great!

File: projects/module-1/library.md (hunk 2 above)
Comment (Kalikoze) [reply projects/module-1/library.md:86]: Agreed, this looks great and appreciate you aligning this with the learning goals of JS Fun at the Restaurant!  Looks ready to be merged!

**Review by Kalikoze — APPROVED** [review]
Looks great to me, thanks for making these updates over the intermission!

**Pull request merged**